src/setup-ganache.ts

```diff
@@ -35,10 +38,11 @@ export default async function(): Promise<string> {
     execArgv: process.execArgv.filter(opt => opt !== '--inspect'),
   });
 
-  const options: Options = {
-    accountsConfig,
+  const options: NodeOptions = {
+    ...config.node,
+    accounts: accountsConfig,
     gasLimit: config.blockGasLimit,
-    gasPrice: config.gasPrice,
+    gasPrice: `0x${config.gasPrice.toString(16)}`,
     coverage: config.coverage,
   };
```

Comment:
I also think this belongs in the `config` module or specifically `getConfig`. I think that module should return something that is ready to forward to Ganache.

Reply:
Yep. That makes sense to isolate all config related stuff to `getConfig`.